Formula/p/periphery.rb

```diff
@@ -15,6 +15,8 @@ class Periphery < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "fdfc5440f8eac20b8290a4173cadfc81090b12ef78454bed09a1a5ef56e30b54"
   end
 
+  depends_on xcode: ["15.4", :build, :test]
+
```

Comment:
with this, then that would affectively drop the ventura support?

Reply:
for the ventura test, you can try with

```
  if MacOS.version <= :ventura
    ENV.prepend_path "PATH", Formula["swift"].opt_bin
  end
```

Reply:
> with this, then that would affectively drop the ventura support?

Yes. I've added `depends_on macos: :sonoma`.